Documentation/spec/docfx_flavored_markdown.md

```diff
@@ -152,3 +152,92 @@ The above content will be transformed to the following html:
   <p>WARNING content</p>
 </div>
 ```
+
+Difference between DFM and GFM
+------------------
+
+### List
+
+DFM list item always uses the first character in the first line to decide whether it is in the list item, but GFM does not.
+For example:
+Case 1:
+```md
+* a
+  * b
+    * c
+
+  text
+```
+
+`a` start at column 3, `b` start at column 5, `c` start at column 7.  
```

Comment:
start -> starts * 3.
Add some explanation to the table? e.g. You can see the number of spaces before "text" will result in different rendered results in DFM and GFM.